File: bookhub-back/src/main/java/com/example/bookhub_back/dto/stock/response/StockLogResponseDto.java
package com.example.bookhub_back.dto.stock.response;

import lombok.Builder;
import lombok.Getter;

import java.time.LocalDate;

@Getter
@Builder
public class StockLogResponseDto {
    private Long stockLogId;
    private String type;
    private String employeeName;
    private String bookTitle;
    private String branchName;
    private Long amount;
    private Long bookAmount;
    private LocalDate actionDate;
    private String description;
}

File: bookhub-front/src/dtos/stock/StockLog.response.dto.ts
import { StockActionType } from "@/apis/enums/StockActionType";

export interface StockLogResponseDto{
    stockLogId: number;
    type: StockActionType;
    employeeName: string;
    bookTitle: string;
    branchName: string;
    amount: number;
    bookAmount: number;
    actionDate: string;
    description: string;
}

